README.md

```diff
@@ -50,3 +54,79 @@ python create_data_set_from_fetched_emails.py
 
 - This will create a file called `dataset.jsonl` in the repo from the contents of the `fetching_state.json` file.
 
+## 5) Running the fine-tuning
+
+### Minimum requirements
+
+Minimum requirements (for Llama fine tuning):
+- 16GB GPU RAM
```

Comment:
Is this sufficient for `Llama-4-Scout-17B-16E`? The model is quite large.

Reply:
no no. 16gb is only sufficient for 8B model. Ill make the readme more clear